docs/FAQ.md

```diff
@@ -30,16 +30,27 @@ to set up testing and validation on their distributions.
 
 ### What CUDA versions are supported?
 
-CUDA 10.1, 10.2 and 11.0 are currently supported, but you need to download the cudf jar that 
-corresponds to the version you are using. Please look [here](download.md) for download 
-links for the latest release.
+CUDA 11.0 and 11.2 are currently supported.  Please look [here](download.md) for download links for
+the latest release.
+
+### What hardware is supported? 
+
+The plugin is tested and supported on V100, T4, A30 and A100 Datacenter GPUs.  It is possible to run
+the plugin on GeForce desktop hardware.  GeForce hardware does not support CUDA enhanced
```

Comment:
Suggest making `CUDA enhanced compatibility` a hyperlink to https://docs.nvidia.com/deploy/cuda-compatibility/index.html#enhanced-compat-minor-releases.  Also we should qualify the GeForce statement with Pascal or later (or perhaps Volta or later), as we can't run on _any_ GeForce hardware.